.github/workflows/tests.yml

```diff
@@ -181,10 +188,37 @@ jobs:
           php-version: ${{ matrix.php-versions }}
           coverage: xdebug
 
-      # Make sure that an nginx configuration file exists in this repository at tests/nginx/php-x.x.conf.
-      # Refer to an existing .conf file in this repository if you need to create a new one e.g. for a new PHP version.
-      - name: Copy nginx configuration file
-        run: sudo cp ${{ env.PLUGIN_DIR }}/tests/nginx/php-${{ matrix.php-versions }}.conf /etc/nginx/conf.d/php-${{ matrix.php-versions }}.conf
+      # Configure nginx to use the PHP version and WOrdPress installation at /var/www/html
```

Comment:
Moved configuring nginx to within the GitHub action, so we don't need to include `tests/nginx/*.conf` files.